pybamm/discretisations/discretisation.py

```diff
@@ -1214,43 +1214,42 @@ def check_variables(self, model):
                         )
                     )
 
-    def search_for_independent_var(self, model, var):
+    def search_for_independent_var(self, var, all_vars_in_eqns):
```

Comment:
I am a bit confused by the function names. Doesn't this one actually check if a given variable is independent, and the other one searches for independent variables in the model?

Reply:
Are the new names better?

Reply:
Yes, thanks!